tests/test_util.py

```diff
@@ -451,33 +451,29 @@ def test_complex_conditions_expr(self, dataset):
                 tuplify(
                     [
                         "if",
-                        [
-                            ["in", ["release", "tuple", ["'foo'"]]],
-                            "release",
-                            "'other'",
-                        ],
-                        "release",
+                        [["in", ["val", "tuple", ["'foo'"]]], "val", "'other'"],
+                        "val",
                     ]
                 ),
```

Comment:
This is to avoid the alias collision issue we discussed today now that `release` is part of the (aliased) columns.